content/getting_started/UsingEnzyme.md

```diff
@@ -120,9 +127,15 @@ We can then compile this to a final binary as follows:
 clang output_opt.ll -o a.exe
 ```
 
-For ease, we could combine the final optimization and bianry execution into one command as follows.
+For ease, we could combine the final optimization and binary execution into one command as follows.
+```sh
+clang output.ll -O2 -o a.exe
+```
+
+These individual steps can be summarised into a single compilation step, where we load Enzyme as a plugin to our compiler clang, and perform the differentiation on our test code.
+
 ```sh
-clang output.ll -O3 -o a.exe
+clang test.c -O2 -Xclang -load -Xclang /path/to/Enzyme/enzyme/build/Enzyme/ClangEnzyme-<VERSION>.so -fno-vectorize -fno-slp-vectorize -fno-unroll-loops -o a.exe
```

Comment:
I might rotate this and start by describing the clang path, and then discussing the internals as what's happening

Reply:
Yes, that would definitely be an option. I think we should just make it more apparent that the steps explained as of now are basically a breakdown of the compilation steps when Enzyme is used as a compiler plugin, as in the call on line 138.